motr/io_req_fop.c

```diff
@@ -181,6 +181,7 @@ static void io_bottom_half(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 		m0_reqh_service_ctx_from_session(reply_item->ri_session),
 		&ioo->ioo_obj->ob_entity, op, &rw_reply->rwr_mod_rep.fmr_remid);
 
+	actual_bytes = rw_reply->rwr_count;
```

Comment:
For optimization, we can avoid actual_bytes getting initialized to 0 as we are setting it here..  
Compiler may do it also..

Reply:
I think "actual_bytes" only get re-assigned to this value in success path.
In failure, it is still "0".
@upendrapatwardhan please check this.

Reply:
Yes, "actual_bytes" only get re-assigned to this value in success path.